src/game/default/g_main.c

```diff
@@ -796,21 +799,23 @@ static void G_Frame(void) {
 			return;
 		}
 	}
+		
+	if (!TIMEOUT) {
+		// treat each object in turn
+		// even the world gets a chance to think
+		g_entity_t *ent = &g_game.entities[0];
+		for (uint16_t i = 0; i < ge.num_entities; i++, ent++) {
+
+			if (!ent->in_use)
+				continue;
 
-	// treat each object in turn
-	// even the world gets a chance to think
-	g_entity_t *ent = &g_game.entities[0];
-	for (uint16_t i = 0; i < ge.num_entities; i++, ent++) {
-
-		if (!ent->in_use)
-			continue;
-
-		g_level.current_entity = ent;
+			g_level.current_entity = ent;
 
-		if (ent->client) {
-			G_ClientBeginFrame(ent);
-		} else {
-			G_RunEntity(ent);
+			if (ent->client) {
+				G_ClientBeginFrame(ent);
```

Comment:
Likewise, not acknowledging / running client commands during `TIMEOUT` is also probably dangerous (?)

If you see weirdness in testing this, we might want to explore freezing clients and `func_*` in place during `TIMEOUT`, rather than skipping them altogether.